anomalous-tx-value/package.json

```diff
@@ -17,11 +27,10 @@
   },
   "dependencies": {
     "bignumber.js": "^9.0.1",
-    "forta-agent": "^0.0.6"
+    "forta-agent": "^0.1.15",
+    "forta-agent-tools": "^3.1.2"
   },
   "devDependencies": {
```

Comment:
I got a `test suite failed to run` error when running `npm test` that was solved by installing back the `@types/jest`

Reply:
something weird happened with the package.json. I'll add @types/jest again